specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -395,6 +395,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/getGitHubAccessToken": {
```

Comment:
Do you expect this action to be used in an ARM template? If so you may want to change the name so it begins with list, if you do this the action will be able to be invoked from a template , you also need to make changes to your manifest to enable this functionality as there is data in the POST body.

Reply:
No, this will not be used in ARM templates.